root/static/scripts/common/components/EntityLink.js

```diff
@@ -411,9 +411,18 @@ $ReadOnlyArray<Expand2ReactOutput> | Expand2ReactOutput | null => {
         />,
       );
     }
-    if (comment) {
+    const primaryAlias = entity.primaryAlias ?? '';
+    if (comment || primaryAlias) {
+      const textParts = [];
+      if (primaryAlias) {
+        textParts.push(primaryAlias);
+      }
+      if (comment) {
+        textParts.push(comment);
+      }
+      const text = textParts.join(', ');
```

Comment:
For a localized comma-separated list, import our `commaOnlyList` and use it as follows:
```suggestion
      const text = commaOnlyList(textParts);
```

Reply:
Thanks! I suspect that `commaOnlyList` can't be used now that I'm italicizing the alias (and the comma after it), but let me know if I'm wrong about that. Maybe I could do some hack like calling `commaOnlyList([alias, ' '])` and then putting that inside the `<i>`.